### DISCUSSION TITLE
Documentation for Table.from_list

### DISCUSSION THREAD

**Discussion by dhilst** [biolab/orange3#6658] [General]
##### Orange version

'3.36.2'

##### Expected behavior

I would like to know how to use `Table.from_list` method or if is there an alternative for creating a table from a lists of lists, or list of dicts.

The domain has discrete and string (meta) variables, but I have no idea how the domain relates to the list of values. Should discrete variables come first? later?

##### Steps to reproduce the behavior

I'm assuming that you have a virtual environment where `orange-canvas` works, and that it is activated

* Clone my for of demo widget `git clone https://github.com/dhilst/orange3-example-addon.git`
* Checkout the commit `git checkout 606b96aafab1ccfae3c7dad3bda8f74fb904107a` (I may update the code)
* Install it `pip install -e orange3-example-addon`
* Start orange `orange-canvas`
* Create a simple workflow with Hello widget and data table to see the results
  <img width="354" alt="image" src="https://github.com/biolab/orange3/assets/392820/c7ad11da-a0b6-4613-ab16-c8bafd660743">
* Click the hello widget, edit the text and press enter
* Open the data table widget
* Orange crashes
  <img width="1280" alt="image" src="https://github.com/biolab/orange3/assets/392820/4e8e790e-7e40-40ec-b873-7b0c24589bb6">

**Comment by markotoplak** [comment]
The best way is to go through `from_numpy`: https://orange3.readthedocs.io/projects/orange-data-mining-library/en/latest/tutorial/data.html#creating-a-data-table

**Reply by dhilst** [reply]
I update the code with from_numpy and it works, thank you!
